moldesign/helpers/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .helpers import *
+from .pdb import *
```

Comment:
Yeah, I'm 99% sure this is the exception to that rule - MDT actually uses this pattern a lot to manage its namespaces. It's definitely worth reviewing how that's done at some point.